compiler/README.md

```diff
@@ -0,0 +1,56 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
+
+# Introduction
+
+This document discusses the ONE compiler. ONE compiler analyses, quantizes and optimizes the trained model. It takes many model formats and converts to a common 'Circle' format which is then deployed. This compiler runs on the target device and targets supported are ExynosXXX.
+
+<!-- Contents -->
+# Contents 
+* [Build Setup](#Build-Setup)
+* [Supported Formats](#Supported-Nerual-Network-Formats)
+* [Model Analysis](#Model-Analysis)
+* [Data Representation & Quantization](#Data-Representation-And-Quantization)
+* [Circle Format](#Circle-Format)
+* [Supported Targets](#Supported-Targets)
+
+<!-- Build Setup -->
+## Build Setup
+<!-- System Requirments -->
+* System Setup   
+This setup is to build the ONE compiler on Ubuntu 18.04.   
+Get the code [Samsung ONE](https://github.com/Samsung/ONE) from Github. 
+
+<!-- Software/Package Requirements -->
+* Software/Package Tested On
+    * Cmake (3.10.2)
+    * g++ toolchain (9.0.1)
+
+<!-- Build Configuration -->
+* Configure  
+In-Progress
```

Comment:
I think you already know it, just a note:

Project is built by cmake/make.
You can use wrappers `./nncc configure` to run cmake and `./nncc build -j$(nproc)` to run make.

Reply:
@binarman Thanks, got it.